src/theory/quantifiers/sygus/sygus_unif_rl.cpp

```diff
@@ -547,6 +560,50 @@ Node SygusUnifRl::DecisionTreeInfo::buildSol(Node cons,
   unsigned c_counter = 0;
   // do we need to resolve a separation conflict?
   bool needs_sep_resolve = false;
+  if (options::sygusUnifCondIndependent())
```

Comment:
Does this block of code belong here?  Maybe we should have buildSol call two independent functions (buildSolUnif and buildSolUnifIndependent or something).  This would allow us to document the methods independently and would be easier to further modularize later.